app/widget.js

```diff
@@ -60,14 +60,14 @@ function renderWidget(el) {
         data: window._SimplechartWidgetData[el.id],
       }));
     }
-
-    // And listen for 'widgetData' event triggered on widget element
-    listenerWidgetData(el, store.dispatch);
   } else {
-    // Bye.
+    // Must have either data-url or data-var
```

Comment:
Probably should add a todo here to actual implement throwing an error, unless that doesn't matter, then this `else` block should be removed.